MiniSim/Extensions/UserDefaults+Configuration.swift

```diff
@@ -8,40 +8,40 @@
 import Foundation
 
 extension UserDefaults {
-    public struct Keys {
+    public enum Keys {
```

Comment:
If we change it to `enum` shouldn't we have cases there? Can you provide some context behind this change? I want to get better understanding of it

Reply:
Not really. It's common practice to use `enum` as namespace. For example, Apple does it in their Combine framework.

Reply:
Sounds good